### PR TITLE
style: various homepage stying and layout fixes

### LINKED ISSUE #745: [TASK]: Loading state for home screen project and paper cards
Currently when we load the home page we see an awkward-looking screen for several seconds until the projects and paper data loads:
image

We can improve the experience by adding an empty/loading state to the project and paper cards, like this:
image

Here is a library for creating the placeholder rectangles with a slight shimmer effect:
https://github.com/michalsnik/vue-content-placeholders

### LINKED ISSUE #748: [TASK]: Add an empty state placeholders to "My projects" and "Shared projects" on the home screen
Currently if the "My projects" and "Shared projects" sections are empty the space reserved for them collapses and the page looks weird.
image

PROPOSED
We need empty state placeholders for both sections. Here's what I propose:
** MY PROJECTS ** 
image

** SHARED PROJECTS **
image

ASSETS
Here's the dot background that can be tiled:
dotPattern  <---- It's tiny and subtle, but it's there

Here's the seed SVG for the "My projects screen":
emptySeed

Here's the plants SVG for the "Shared projects screen":
emptyPlants

### LINKED ISSUE #751: [BUG]: Make paper and project cards on the home screen all the same height
Currently the paper cards on the home screen aren't all the same size. For instance, if the author field is missing from the paper card, the card is shorter.
Monosnap Terarium 2023-02-21 14-49-46

PROPOSED
The cards should have a fixed height. All cards should be the same height.

While you're at this, let's see if we can tighten up some of the spacing in the cards as well. Here are some specs for reference. Feel free to use rems or some other approach if specifying pixel heights doesn't feel right to you:

PAPER CARDS
image

PROJECT CARDS
image

### PR DESCRIPTION
**Pull request by echl**
# Description

* Add loading skeletons for project and document cards
* Display placeholders for "My Projects" and "Shared Projects" tabs when there are no projects to display.
* Resize paper and document cards
* Add highlight to carousel on hover and increase click area, reduce carousel scroll speed
* Add new project card to end of project carousel

https://user-images.githubusercontent.com/16358728/223845413-7d058a1e-4766-4194-8a9a-3544ee748181.mov

https://user-images.githubusercontent.com/16358728/223845771-3e26dd05-35df-4f06-a959-bef55f7d6cf0.mov

https://user-images.githubusercontent.com/16358728/223846299-e6370306-fd7e-4313-95ef-6b55416509c5.mov

https://user-images.githubusercontent.com/16358728/223846460-3fa6ac01-1967-4ea8-960d-9e1f0aff0ab7.mov

Resolves #745
Resolves #748
Resolves #751

### CODE REVIEW

File: packages/client/hmi-client/src/page/Home.vue [hunk 1]
```diff
@@ -54,52 +82,64 @@
...
+						:selected-document="selectedDocument"
+						@close="close()"
+					/>
 				</div>
-				<selected-document-pane
-					class="selected-document-pane"
-					:selected-document="selectedDocument"
-					@close="close()"
-				/>
 			</div>
```
Comment (YohannParis) [inline packages/client/hmi-client/src/page/Home.vue:115]: This part I'm confused, why not use the `<Modal>` component here?

File: packages/client/hmi-client/src/assets/css/theme/_variables.scss [hunk 2]
```diff
@@ -904,4 +904,5 @@ $imagePreviewActionIconBorderRadius: 50% !default;
 	--focus-ring: #{$focusShadow};
 	--maskbg: #{$maskBg};
 	color-scheme: light;
+	--chevron-hover: #8bd4af1a;
```
[the remark this answers is not in the document]
Comment (echl) [reply packages/client/hmi-client/src/assets/css/theme/_variables.scss:907]: Not picky at all, you're absolutely right!

File: packages/client/hmi-client/src/components/projects/ProjectCard.vue [hunk 3]
```diff
@@ -3,13 +3,14 @@ import { IProject, ProjectAssetTypes } from '@/types/Project';
...
 
-defineProps<{ project: IProject }>();
+defineProps<{ project?: IProject }>();
 </script>
 
 <template>
-	<Card>
-		<template #header>
+	<Card v-if="project">
+		<template #content>
```
[the remark this answers is not in the document]
Comment (echl) [reply packages/client/hmi-client/src/components/projects/ProjectCard.vue:13]: Yeah, exactly.

File: packages/client/hmi-client/src/page/Home.vue [hunk 4]
```diff
@@ -54,52 +82,64 @@
 						</ul>
 					</div>
 				</div>
+				<div v-if="isLoadingProjects">
+					<p>
+						<Skeleton width="6rem" />
```
[the remark this answers is not in the document]
Comment (echl) [reply packages/client/hmi-client/src/page/Home.vue:87]: This is how it's suggested to be used in the documentation!

File: packages/client/hmi-client/src/page/Home.vue (hunk 1 above)
Comment (echl) [reply packages/client/hmi-client/src/page/Home.vue:115]: This wasn't touched by this PR, just moved around.

File: packages/client/hmi-client/src/page/Home.vue [hunk 5]
```diff
@@ -54,52 +82,64 @@
...
+					</p>
+					<div class="carousel">
+						<ul>
+							<li v-for="(i, index) in [0, 1, 2, 3, 4, 5]" class="card" :key="index">
```
[the remark this answers is not in the document]
Comment (echl) [reply packages/client/hmi-client/src/page/Home.vue:91]: Looks like I can get rid of that class and corresponding styling completely! It must have been a leftover from the old design.

File: packages/client/hmi-client/src/components/projects/ProjectCard.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (echl) [reply packages/client/hmi-client/src/components/projects/ProjectCard.vue:13]: Much easier to implement jamie's designs without using the existing slots.

File: packages/client/hmi-client/src/components/documents/DocumentCard.vue [hunk 6]
```diff
@@ -62,26 +83,26 @@ function getRandomImage() {
...
-	const doi = await getDocumentDoi(props.document);
-	await fetchArtifacts(doi);
-	if (artifacts.value.length > 0) {
-		extractionType.value = artifacts.value[0].askemClass;
+	if (props.document) {
+		const doi = await getDocumentDoi(props.document);
+		await fetchArtifacts(doi);
+		if (artifacts.value.length > 0) {
+			extractionType.value = artifacts.value[0].askemClass;
+		}
```
[the remark this answers is not in the document]
Comment (echl) [reply packages/client/hmi-client/src/components/documents/DocumentCard.vue:91]: Looking at it, I would even put all of `fetchArtifacts` inside `onMounted`, I don't see why it needs to be its own method

File: packages/client/hmi-client/src/page/Home.vue [hunk 7]
```diff
@@ -204,6 +204,7 @@ onMounted(async () => {
 	queryStore.reset(); // Facets queries.
 
 	projects.value = (await API.get('/home')).data as IProject[];
+	projects.value = projects.value.slice(1);
```
Comment (YohannParis) [inline packages/client/hmi-client/src/page/Home.vue:207]: What is this?

Comment (echl) [reply packages/client/hmi-client/src/page/Home.vue:207]: whoops, meant to delete

**Review by YohannParis — DISMISSED** [review]

**Review by YohannParis — DISMISSED** [review]

**Review by YohannParis — APPROVED** [review]

**Pull request merged**